src/Runner.Worker/Handlers/NodeScriptActionHandler.cs

```diff
@@ -102,6 +103,11 @@ public async Task RunAsync(ActionRunStage stage)
                 Data.NodeVersion = "node16";
             }
 #endif
+            string nodeVersionOut = NodeUtil.GetNodeVersion();
```

Comment:
```suggestion
            var forcedNodeVersion = System.Environment.GetEnvironmentVariable(Constants.Variables.Agent.ForcedActionsNodeVersion);
```

Let's remove the method above above